packages/atlas/src/components/_comments/CommentInput/CommentInput.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import { Meta, Story } from '@storybook/react'
+import React from 'react'
+import { MemoryRouter } from 'react-router-dom'
+
+import { CommentInput, CommentInputProps } from './CommentInput'
+
+export default {
+  title: 'comments/CommentInput',
+  component: CommentInput,
+  args: {
+    cancelButton: false,
+  },
+  decorators: [
+    (Story) => (
+      <MemoryRouter>
+        <Story />
+      </MemoryRouter>
+    ),
+  ],
+  argTypes: {},
+} as Meta<CommentInputProps>
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const Template: Story<CommentInputProps & any> = (args) => (
```

Comment:
Why do we need `any` here?

Reply:
Maybe we could also add `description` to this particular control? I don't know how it works in a storybook - this is not super important,  just a suggestion.